pybrowscap/loader/csv/__init__.py

```diff
@@ -16,7 +16,7 @@
 
 
 # Url where latest version of csv browscap data file is located
-URL = 'http://browscap.org/stream?q=BrowsCapCSV'
+URL = 'http://browscap.org/stream?q=BrowsCapINI (replace BrowsCapINI with the appropriate version)'
```

Comment:
You change it here from csv to ini. Is there a reason ? Also I would remove ` (replace BrowsCapINI with the appropriate version)` because without this the URL is not downloadable in latest version; with it - NO

Reply:
oh, this is a mistake.